src/metadata/key_descriptor.rs

```diff
@@ -11,13 +11,17 @@ const NAME: &str = "md:KeyDescriptor";
 pub struct KeyDescriptor {
     #[serde(rename = "use")]
     pub key_use: Option<String>,
-    #[serde(rename = "ds:KeyInfo")]
+    #[serde(rename = "KeyInfo")]
     pub key_info: KeyInfo,
-    #[serde(rename = "md:EncryptionMethod")]
+    #[serde(rename = "EncryptionMethod")]
     pub encryption_methods: Option<Vec<EncryptionMethod>>,
 }
 
 impl KeyDescriptor {
+    pub fn is_signing(&self) -> bool {
+        self.key_use == Some("signing".to_string())
```

Comment:
```suggestion
        if let Some(key_use) = self.key_use.as_ref().map(String::as_str) {
            return key_use == "signing";
        }
        false
```
Instead of allocating another string for this comparison, lets use a static string and dereference the option.